src/libraries/System.Net.Sockets/tests/FunctionalTests/SendFile.cs

```diff
@@ -4,6 +4,7 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Runtime.Versioning;
```

Comment:
Was this intentional?

Reply:
Nope.  No idea why that got added.  VS by default now automatically adds namespaces for things; I probably typed something that happened to coincide with something in that namespace.

Reply:
Yeah, that happens to me all the time now.